README.md

```diff
@@ -13,32 +13,60 @@ Describe your example and specify what it is demonstrating. Consider the
 following questions:
 
 * Is it pedagogical or a usable application?
-* Does it belong to a particular domain such as material simulation or logistics? 
-* What level of Ocean proficiency does it target (beginner, advanced, pro)? 
+* Does it belong to a particular domain such as material simulation or logistics?
+* What level of Ocean proficiency does it target (beginner, advanced, professional)?
 
 A clear description allows us to properly categorize your example.
 
 Images are encouraged. If your example produces a visualization, consider
 displaying it here.
 
-![D-Wave Logo](dwave_logo.png)
+![D-Wave Logo](static/dwave_logo.svg)
 
-## Usage
+## Installation
+You can run this example without installation in cloud-based IDEs that support the [Development Containers specification](https://containers.dev/supporting) (aka "devcontainers").
 
-A simple command that runs your program. For example,
+For development environments that do not support `devcontainers`, install requirements:
 
 ```bash
-python <demo_name>.py
+pip install -r requirements.txt
 ```
 
+If you are cloning the repo to your local system, working in a [virtual environment](https://docs.python.org/3/library/venv.html) is recommended.
+
+## Usage
+Your development environment should be configured to [access Leap’s Solvers](https://docs.ocean.dwavesys.com/en/stable/overview/sapi.html). You can see information about supported IDEs and authorizing access to your Leap account [here](https://docs.dwavesys.com/docs/latest/doc_leap_dev_env.html).
+
 ### Inputs
 If your example requires user input, make sure to specify the input format and any input limitations.
 ### Outputs
 An example program output.
 
+## Demo Types
+### User Interface Demo (Dash Framework Template)
+The Dash template is intended for demos that would benefit from a user interface. This user interface could include settings to run and customize the problem, an interactive graphical element, and/or tables or charts to compare different solutions. This template is also useful for demos that are intended for a general audience, as it is more approachable for those without a technical background.
```

Comment:
For text like this we really need a way to distinguish two types of content in this template:
1. Explanations of the template directed at the contributor 
2. Content that should be in the new example's README

The way we do this in PS templates for customer reports is with color but GitHub support for colored text is [complicated](https://github.com/orgs/community/discussions/31570). Perhaps a standard sentence placed before content that should be in the example README

Reply:
Yeah I was struggling with this too while writing it. Do you think it would be helpful to separate the two sections into two top level headers that describe that one is a template to fill out and the other is a demo creator manual can be deleted after the demo is done? That way the creator can easily remove one of the two sections and everything is in one place?

Reply:
That sounds like what might be a good approach. It's a bit hard to know without looking at the result but I think it might work well.

Reply:
I was thinking about this as well, and I think separating them out into two very distinct sections works as a solution, another one being having a separate setup-instructions file. Not sure which I like best.

Reply:
Ok sounds good I'll give it a shot and you can let me know if you think it's good!